src/email.c

```diff
@@ -151,7 +151,7 @@ handle_compose_email (XdpImplEmail *object,
   g_variant_lookup (arg_options, "attachments", "^a&s", &attachments);
 
   if (!compose_mail (address, subject, body, attachments, NULL))
-    response = 1;
+    response = 2;
```

Comment:
We're bound by the portal api, which just returns an uint as status. Furthermore, we don't really want to leak information into the sandbox about what went wrong...